runtime/worker.rs

```diff
@@ -1084,6 +1089,7 @@ fn common_extensions<
     ops::permissions::deno_permissions::init(),
     ops::tty::deno_tty::init(),
     ops::http::deno_http_runtime::init(),
+    ops::bundle::deno_bundle_runtime::lazy_init(),
```

Comment:
Why is it `lazy_init()` here but `init()` in `web_worker.rs`?

Reply:
Not entirely sure, I was matching the rest of the extensions that have args (for instance deno_node and a bunch of others are the same)